addons/test_event_full/static/src/js/tours/wevent_register_tour.js

```diff
@@ -22,6 +22,12 @@ var discoverTalkSteps = function (talkName, fromList, reminderOn, toggleReminder
             run: 'click',
         }];
     }
+    steps = steps.concat([{
+        content: "Check we are on the talk page",
+        trigger: 'div.o_wesession_track_main',
+        run: function () {}, // it's a check
+    }]);
+
     if (reminderOn) {
         steps = steps.concat([{
             content: "Check Favorite is on",
```

Comment:
As really we shouldn't be depending on case,
```suggestion
            content: `Check Favorite for ${talkName} was already on`,
```
And for line L45/R47,
```suggestion
            content: `Check Favorite for ${talkName} is now On`,
```